pkg/apis/kubeflow.org/v1/paddlepaddle_types.go

```diff
@@ -0,0 +1,135 @@
+// Copyright 2020 The Kubeflow Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1
+
+import (
+	commonv1 "github.com/kubeflow/common/pkg/apis/common/v1"
+	autoscalingv2beta2 "k8s.io/api/autoscaling/v2beta2"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	// PaddleJobDefaultPortName is name of the port used to communicate between Master and
+	// workers.
+	PaddleJobDefaultPortName = "master"
+	// PaddleJobDefaultContainerName is the name of the PaddleJob container.
+	PaddleJobDefaultContainerName = "paddle"
+	// PaddleJobDefaultPort is default value of the port.
+	PaddleJobDefaultPort = 36543
+	// PaddleJobDefaultRestartPolicy is default RestartPolicy for PaddleReplicaSpec.
+	PaddleJobDefaultRestartPolicy = commonv1.RestartPolicyOnFailure
+	// PaddleJobKind is the kind name.
+	PaddleJobKind = "PaddleJob"
+	// PaddleJobPlural is the PaddlePlural for paddleJob.
+	PaddleJobPlural = "paddlejobs"
+	// PaddleJobSingular is the singular for paddleJob.
+	PaddleJobSingular = "paddlejob"
+	// PaddleJobFrameworkName is the name of the ML Framework
+	PaddleJobFrameworkName = "paddle"
```

Comment:
Do we need this const?

Reply:
Thx. 
Yes, they are referred in the expectation and logger part.